M2 registration.php.php

```diff
@@ -1,6 +1,6 @@
 <?php
 \Magento\Framework\Component\ComponentRegistrar::register(
-    \Magento\Framework\Component\ComponentRegistrar::MODULE,
+    \Magento\Framework\Component\ComponentRegistrar::${Component_Type},
```

Comment:
Can you make it so that it defaults to `MODULE` if not specified? Like here: https://github.com/lfolco/phpstorm-m2-filetemplates/blob/master/M2%20composer.json.json#L5